pk-test/docker.lua

```diff
@@ -0,0 +1,191 @@
+--------------------------------------------------------------------------------
+-- docker.lua
+-- This file is a part of pk-test library
+-- Copyright (c) pk-test authors (see file COPYRIGHT for the license)
+-- See file `COPYRIGHT` for the license
+--------------------------------------------------------------------------------
+
+require 'lua-aplicado.module' -- for import()
```

Comment:
The proper way to say that is `require 'lua-nucleo'`.

However, this tool implicitly depends on le-run-lua-module, which already makes import available.

Please, either change the whole pk-test codebase to explicitly depend on lua-nucleo, or drop this line.